expo/Greedy.py

```diff
@@ -0,0 +1,19 @@
+import random
+
+from expo.MCTS import MCTS
```

Comment:
It needs to be placed in a suitable place and the name updated.

Reply:
done